test/src/org/exist/xmldb/XmldbURITest.java

```diff
@@ -13,7 +13,7 @@
 
 public class XmldbURITest {
 
-    private final static String jettyPort = System.getProperty("jetty.port");
+    private final static String jettyPort = System.getProperty("jetty.port", "8080");
```

Comment:
This doesn't need to be a `System.getProperty`. This test has no real reliance on Jetty.  Instead I would suggest:

```java
private final static int PORT = 8080;
```

Although it is arguable that you even need it as a global constant.